samples/boards/intel_s1000_crb/audio/audio_app.rst

```diff
@@ -7,15 +7,17 @@ Overview
 ********
 
 This sample application demonstrates audio capture and playback on an
-Intel® S1000 CRB.
+Intel® Sue Creek S1000 CRB with control from a Linux host connected
+over USB.
 The application uses the following drivers
 
 - DMIC driver (For microphone audio capture)
 - I2S driver (For audio input from host,
   audio output to host and audio output to codec)
 - Codec driver
+- USB driver and USB HID stack
 
-Intel® S1000 is the master on the I2S interfaces.
+Intel® Sue Creek is the master on the I2S interfaces.
```

Comment:
Legal names can't be abbreviated, so this should be ``Intel® Sue Creek S1000``

Reply:
sure. i don't mind using the looong name everywhere -- hard to read, but will make legal happy.